doc/ChangeLog

```diff
@@ -1,4 +1,132 @@
 ===============================================================
+Tag name:  ctsm5.1.dev022
+Originator(s):  gregorylemieux (Gregory Lemieux,LBL/NGEET,510-486-5049)
+Date:  Wed 27 Jan 2021 12:26:44 AM PST
+One-line Summary: Merge fates_main_api into ctsm master
+
+Purpose and description of changes
+----------------------------------
+
+This brings fates_main_api into and up-to-date with ctsm master.
+At the point of merge, fates_main_api was parallel with ctsm5.1.dev020,
+only one tag behind ctsm master's head.
+
+The purpose of this merge is to make all future updates to fates
+on the ctsm master branch instead of on the fates_main_api branch.
+
+
+Significant changes to scientifically-supported configurations
+--------------------------------------------------------------
+
+Does this tag change answers significantly for any of the following physics configurations?
+(Details of any changes will be given in the "Answer changes" section below.)
+
+    [Put an [X] in the box for any configuration with significant answer changes.]
+
+[ ] clm5_1
+
+[X] clm5_0
+
+[ ] ctsm5_0-nwp
+
+[X] clm4_5
+
+
+Bugs fixed or introduced
+------------------------
+[Remove any lines that don't apply. Remove entire section if nothing applies.]
+
+Issues fixed (include CTSM Issue #): none
+
+CIME Issues fixed (include issue #): none
+
+Known bugs introduced in this tag (include issue #): none
+
+Known bugs found since the previous tag (include issue #): none
+
+
+Notes of particular relevance for users
+---------------------------------------
+[Remove any lines that don't apply. Remove entire section if nothing applies.]
+
+Caveats for users (e.g., need to interpolate initial conditions): none
+
+Changes to CTSM's user interface (e.g., new/renamed XML or namelist
+variables): none
+
+Changes made to namelist defaults (e.g., changed parameter values): none
+
+Changes to the datasets (e.g., parameter, surface or initial files): none
+
+Substantial timing or memory changes: 
+   Only the fates testmod FatesAllVars failed the MEMCOMP test
```

Comment:
Let's talk about this failed test more. Can you document it in comments? I suppose if this just means that the model is using more memory because FATES has more complexity and more variables that's likely expected.

Reply:
Updated per b1a6080a90fa50cc5d9f78efcf2fccc221744cb3